src/collection/components/CollectionFragment.tsx

```diff
@@ -80,8 +77,12 @@ const CollectionFragment: FunctionComponent<CollectionFragmentProps> = ({
 		setUseCustomFields(!useCustomFields);
 	};
 
-	const renderForm = (fragment: any, itemMeta: any, index: number) => {
-		const isVideoBlock = !useCustomFields && itemMeta;
+	const renderForm = (
+		fragment: Avo.Collection.Fragment,
+		itemMetaData: Avo.Item.Response,
+		index: number
+	) => {
+		const isVideoBlock = !useCustomFields && itemMetaData;
```

Comment:
is dit niet genoeg?
const isVideoBlock: boolean = !!itemMetaData;

Reply:
Nee, want anders zijn de custom fields ook disabled als de "Alternatieve velden" toggle aan staat.